### PR TITLE
fix: typo preventing display of assignment name

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** A typo was preventing assignment name from updating correctly

JIRA: [EDUCATOR-5640](https://openedx.atlassian.net/browse/EDUCATOR-5640)

**Developer Checklist**
- [x] Test suites passing
- [x] Received code-owner approving review
- [x] Bumped version number [package.json](../package.json)

**Testing Instructions**

- Running gradebook locally, click on a grade to bring up the edit grade modal.
- The assignment name should be filled in at the top of the modal.

Before:

Screen Shot 2021-03-22 at 2 04 42 PM

After:

Screen Shot 2021-03-22 at 2 05 41 PM

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [x] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Review by jnlapierre — APPROVED** [review]
LGTM

**Review by nsprenkle — APPROVED** [review]
Have verified locally :shipit:
